saleor/checkout/base_calculations.py

```diff
@@ -192,9 +193,12 @@ def base_checkout_line_total(
     )
 
 
-def base_order_line_total(order_line: "OrderLine"):
-    unit_price = order_line.unit_price * order_line.quantity
-    return quantize_price(unit_price, unit_price.currency)
+def base_order_line_total(order_line: "OrderLine") -> OrderTaxedPricesData:
```

Comment:
I think this should be moved to some "order" module since this one is for checkout.